src/queries.jl

```diff
@@ -322,7 +324,7 @@ for (tagsymbol, tagvariant) in pairs(tag_types)
                     end
                 end
             end
-            allB ? res : nothing
+            allB ? res : nothing # do we need to reverse here?
```

Comment:
do not leave things like this in comments, especially not comments that do not have TODO or XXX marked to them, otherwise we will have silent bugs. Stuff like that needs to be fixed before sending for review